src/test/routes/return-to-existing.test.ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from 'chai';
+import request from 'supertest';
+
+import { app } from '../../main/app';
+
+describe('GET /return-to-existing', () => {
+  it('should return the return to existing page', async () => {
```

Comment:
'return the return' looks like a typo. Please clarify the should statement

Reply:
amended